ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/backwardsync/BackwardSyncContextTest.java

```diff
@@ -83,7 +83,7 @@ public class BackwardSyncContextTest {
   public static final int REMOTE_HEIGHT = 50;
   public static final int UNCLE_HEIGHT = 25 - 3;
 
-  public static final int NUM_OF_RETRIES = 100;
+  public static final int NUM_OF_RETRIES = 1;
```

Comment:
only one retry ? in case of bad peers it will not be a problem? not sure how backwardsync is working so just asking

Reply:
This is test code, prod code has 2 retries (which is actually just one retry: two "tries")